code/FBXConverter.cpp

```diff
@@ -1284,6 +1286,8 @@ unsigned int Converter::ConvertMeshSingleMaterial( const MeshGeometry& mesh, con
         if ( uvs.empty() ) {
             break;
         }
+        out_mesh->mNumIndices = std::accumulate(faces.begin(), faces.end(), 0);
+        out_mesh->mIndices = new unsigned int[out_mesh->mNumIndices];
```

Comment:
This line wipes out the existing mIndices (withing freeing) and results in the index buffer being unallocated.